runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkStreamingTranslationContext.java

```diff
@@ -62,20 +62,20 @@ public FlinkStreamingTranslationContext(StreamExecutionEnvironment env, Pipeline
     this.dataStreams = new HashMap<>();
   }
 
-  public StreamExecutionEnvironment getExecutionEnvironment() {
```

Comment:
Instead I would prefer to make the class public, since it is required to implement custom translations. Those are otherwise forced to be placed into the same package. And there are problems with package scope and multiple class loaders.